protected/humhub/modules/file/widgets/ShowFiles.php

```diff
@@ -46,14 +46,14 @@ public function run()
             return;
         }
 
-        $hidePreviewFileInfo = ($this->preview) ? Yii::$app->getModule('file')->settings->get('hideImageFileInfo') : false;
+        $excludeMediaFilesPreview = ($this->preview) ? Yii::$app->getModule('file')->settings->get('excludeMediaFilesPreview') : false;
```

Comment:
@buddh4 , We can add default 1 here.

Reply:
We should either create a migration and transfer the value of the old setting. And/Or create a central getter for this setting with a default value of 1.